Extension/src/githubAPI.ts

```diff
@@ -56,12 +56,12 @@ function isAsset(input: any): input is Asset {
 
 /**
  * Determine whether an object is of type Build. Note that earlier releases of the extension
- * do not have 4 or greater Assets (Mac, Win, Linux 32/64). Only call this on more recent Builds.
+ * do not have 3 or greater Assets (Mac, Win, Linux). Only call this on more recent Builds.
```

Comment:
I think there might actually be 6 assets currently (since the source is zipped with each release), and this will go to 5.  We might want to check that there are 3 assets with a ".vsix" extension.

Reply:
The code gives 4 when I debugged it, i.e. something is choosing the 4 vsix's as assets and ignoring the other stuff.

Reply:
The raw JSON has 4 assets only. See https://api.github.com/repos/Microsoft/vscode-cpptools/releases . The other 2 that appear in the web interface are actually stored in different properties in the raw JSON, and the GitHub UI just presents them as "assets".

Reply:
ok. It's been a while since we wrote this code.